lib/chargebee_rails/webhook_handler.rb

```diff
@@ -215,8 +215,8 @@ def payment_method_attrs customer, card
         auto_collection: customer.auto_collection,
         payment_type: customer.payment_method.type,
         reference_id: customer.payment_method.reference_id,
-        card_last4: card.last4,
-        card_type: card.card_type,
+        card_last4: card&.last4,
```

Comment:
@ConfusedVorlon I see that the safe navigation operator of ruby 2.3 is used here, unfortunately this is not supported in previous ruby versions, so can we use the more generic version of the same (card and card.last4) ?

Reply:
changed